### PR TITLE
Updated test directions and created a Python file that will run all m…

### PR DESCRIPTION
**Pull request by bull313**
…od tests

Issue #15

### CODE REVIEW

File: Tests/Test.txt [hunk 1]
```diff
@@ -1 +1,82 @@
...
+        def test_method2():
+            ...test code...
+
+        def test_method2_overloaded():
+            ..test code...
+
+    Each method should be tested and the function that tests it shall have the same name but with a "test_" prefix just like the example above.
+
+STEP 4: Add the unit test to run_mod_tests.py
+    In order to run the new module test with all other module tests, open run_mod_tests.py directly in the "Tests" directory.
```
Comment (brhumphe) [inline Tests/Test.txt]: It may also be valuable to document how to setup and run unit tests in PyCharm.

Comment (bull313) [reply Tests/Test.txt]: Okay. I'm not super familiar with PyCharm, so I'm not sure how to do that. All I know is that the unit testing module I'm using can be run straight from a console window, which I would imagine also includes a console window in PyCharm by running run_mod_tests.py or running an individual mod test (once we have them)

Comment (brhumphe) [reply Tests/Test.txt]: Pretty much. We can leave it as-is.

**Comment by brhumphe** [comment]
Resolved #15

**Pull request merged**